front/app/components/VerticalCenterer/index.tsx

```diff
@@ -4,7 +4,6 @@ export default styled.div`
   height: 100%;
   flex: 1 0 auto;
   display: flex;
-  flex-direction: column;
```

Comment:
Ironically, this was not a vertical "centerer". It moved its content to the top of the page.